kytos/core/tcp_server.py

```diff
@@ -98,8 +102,10 @@ def setup(self):
         if server_port in self.known_ports:
             protocol_name = self.known_ports[server_port]
         else:
-            protocol_name = f'{server_port:04d}'
-        self.connection.protocol.name = protocol_name
+            protocol_name = self.server.protocol_name
+        # strip() to make sure quotes won't mess up KytosEvent name
+        self.connection.protocol.name = str(protocol_name)\
```

Comment:
Did you faced any problems with quotes?! This wouldn't be a problem, I think.

Reply:
Kind of. This problem only shows up if the user sets the protocol_name in the cfg file with quotes or double quotes. However, if the user doesn't specify a string this isn't a problem. Since there are other strings that don't have quotes in the cfg file I believe the user should follow the same pattern and don't specify quotes.  For example @diraol, let's say:

If I were to set (in the kytos conf file):

`#` The southbound protocol that will be used by the TCP server (note the single quote bellow)
protocol_name = 'openflow'

and if I were to debug the protocol_name variable in the tcp_server, you'd see this, and it would be a problem:
```
2017-10-04 18:47:58,356 - INFO [kytos.core.tcp_server] (Thread-11) protocol_name 'openflow'
```